### DISCUSSION TITLE
Helpdesk odoo17 on containermanager

### DISCUSSION THREAD

**Discussion by sybdeb** [OCA/odoo-community.org#186] [Q&A]
Hello,

i am a noob in programming, used to be teamlead b2c sales and customerservice for a webshop. Introduced Odoo together with a system admin whom had a lot off experience. Recently is lost my job and am trying to get Odoo 17 running as a website with webshop. But i am struggeling al lot with getting the helpdesk installed. it is in the file extra_addons but somehow it will not get installed.

who has the knowledge to get this working, because chatgpt hasnot

greetings Sybren

**Comment by castillocon** [comment]
You have an option that, while not fully comprehensive, does its job and works.

https://apps.odoo.com/apps/modules/17.0/odoo_website_helpdesk
